src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/RazorCSharpLanguageMiddleware.ts

```diff
@@ -4,6 +4,7 @@
  * ------------------------------------------------------------------------------------------ */
```

Comment:
@ajaybhargavb this is your baby, you should review like your life depends on it 🤣